src/Adapter/Form/ChoiceProvider/FeatureValuesChoiceProvider.php

```diff
@@ -74,6 +74,7 @@ public function getChoices(array $options)
         if (isset($options['custom'])) {
             $filters['custom'] = $options['custom'];
         }
+        $filters['id_lang'] = $options['id_lang'] ?? $this->contextLanguageId;
```

Comment:
Fetching it only in one language will prevent the conditions below from working, there are some fallbacks if the entry doesn't exist in that language.

However, I think that there should ALWAYS be a language entry.

Reply:
Yep right, I modified it a bit to fetch potential multiple languages but maximum two